drivers/volume/portworx/portworx.go

```diff
@@ -60,13 +60,12 @@ const (
 	RuleScaleTypeResizeDisk = "resize-disk"
 	// RuleMaxSize is name for rule max size
 	RuleMaxSize = "maxsize"
+	// CsiStorage CSI storage driver name
+	CsiStorage torpedovolume.StorageProvisionerType = "csi"
+	// GkeStorage GKE storage driver name
+	GkeStorage torpedovolume.StorageProvisionerType = "gke"
```

Comment:
These should not be here?

Reply:
moved to their individual driver implementations.